packages/calling/README.md

```diff
@@ -1,52 +1,75 @@
-# @webex/calling
+### Table of Contents
+- [Getting Started](#getting-started)
+- [Developing](#developing)
+- [Building](#building) 
+- [Testing](#testing)
+- [Samples](#samples) 
+- [Consuming SDK](#consuming-sdk)
+  - [NPM](#npm)
+  - [CDN](#cdn)
+- [Kitchen Sink App](#kitchen-sink-app)
+---
 
+## Getting Started
 With the Webex Calling SDK, you can effortlessly integrate fundamental audio calling capabilities into your solutions, enhancing the way your users connect.
 
-> [Introduction to the Webex Web Calling SDK](https://github.com/webex/webex-js-sdk/wiki/Introducing-the-Webex-Web-Calling-SDK)
-> [Quickstart guide](https://github.com/webex/webex-js-sdk/wiki/Quickstart-Guide-(Calling))
-> API guide - TBD
-
+- [Introduction to the Webex Web Calling SDK](https://github.com/webex/webex-js-sdk/wiki/Introducing-the-Webex-Web-Calling-SDK)
+- [Quickstart guide](https://github.com/webex/webex-js-sdk/wiki/Quickstart-Guide-(Calling)).
+- API guide - TBD.
```

Comment:
This is the API guide. I don't think we need this line

Reply:
Removed